Makefile

```diff
@@ -71,7 +71,7 @@ verify: verify-scripts verify-codegen-crds
 
 local-deploy-manifests:
 	kubectl apply -f ./manifests
-	kubectl apply -f ./bundle/manifests
+	kubectl apply $$(ls ./bundle/manifests/*.crd.yaml | awk ' { print " -f " $$1 } ')
```

Comment:
suggestion: maybe let's put the cluster service version or the CRDs in its own directory?

Reply:
We'd need to move it around in the downstream then. It's not nice that CRDs and expected to coexist with CSVs in bundles :face_exhaling:

Reply:
ah ... ugh, that's a strange constraint.